source/usage-examples/command.txt

```diff
@@ -5,23 +5,33 @@ Run a Command
 .. default-domain:: mongodb
 
 You can run all raw database operations, not including
-:manual:`CRUD </crud/>` operations, using the :node-api:`command()
-<Admin.html#~command>` method. Call the ``command()`` method with
-your command object on an instance of ``Admin`` for diagnostic and
+:manual:`CRUD </crud/>` operations, using the :node-api:`db.command()
+<Db.html#~command>` method. Call the ``command()`` method with
+your command object on an instance of a database for diagnostic and
 administrative tasks such as fetching server stats or initializing a replica
 set.
 
 .. note::
     Use :manual:`collection methods </reference/method/js-collection>`
     instead of raw database commands whenever possible.
 
-Create an object to specify options. Set the ``maxTimeMS`` field of this
-object to state the number of milliseconds to wait before terminating the
-query.
+You can specify additional options in the ``options`` object passed in
+the second parameter of the ``deleteMany()`` method. For more information
+on the options you can pass, see the
+:node-api:`bb.command() API documentation <Db.html#~command>`. You can
+also pass a :node-api:`callback method <Admin.html#~resultCallback>` as an
+optional third parameter.
 
-The ``command()`` method returns a :mdn:`Promise
-<Web/JavaScript/Reference/Global_Objects/Promise>`. The Promise returns
-the results of the command operation when it completes.
+The ``command()`` method behaves in two different ways depending on whether
```

Comment:
I'd like to restate my desire to handle all documentation of the `Promise` vs `callback` approach in a single document at the _top_ of the driver documentation. Every top-level API method of the node driver (with very few exceptions) will return a `Promise` if no `callback` is provided. We shouldn't have to remind users of that at every step, they can read it in one place (with rationale), and then see it in the API documentation.

Reply:
This sounds possible; I'll have to check with the DevEd team to get their opinions since we've specifically ordered the information to help both new developers and full-stack developers understand what the method does before providing them more in-depth information about the return types.

Reply:
I'll add a `note::` directive that contains this info and delegate mention of NodeJS driver-specific return types to the API. I think the team still prefers to identify the type the Promise resolves to, but we can revisit that another time.